pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-configuration</artifactId>
-  <version>5.8.0-SNAPSHOT</version>
+  <version>5.8.1-SNAPSHOT</version>
```

Comment:
Version 5.8.0 is the next version and has not been released. Why are you changing that version?

Reply:
Fixed